src/components/registration-panel/registration-panel.scss

```diff
@@ -39,6 +39,20 @@
       margin-bottom: 50px;
     }
 
+    // all registration buttons
+    .registration-button {
+      border-color: $white;
+      color: $white;
+      padding: 10px;
+
+      &:not(.disabled) {
```

Comment:
didn't know `:not()` was a thing. That is pretty dope

Reply:
yesss, there's a bunch of niche pseudo selectors but this is one of the more underrated ones